src/app/cup/cup-rating-group/cup-rating-group.component.html

```diff
@@ -38,155 +38,8 @@ <h4>Рейтинг групи</h4>
-                     <td>{{ cupRatingGroupItem.loss }}</td>
-                  </tr>
-               </table>
-            </td>
-            <td class="text-center align-middle">{{ cupRatingGroupItem.scored }}:{{ cupRatingGroupItem.missed }}</td>
-            <td class="text-center align-middle">
-               <strong>{{ cupRatingGroupItem.points }}</strong>
-            </td>
-            <td class="text-center align-middle">
-               <table class="table-sm text-muted" align="center">
-                  <tr>
-                     <td>
-                        <small>{{ cupRatingGroupItem.points_sum }}</small>
-                     </td>
-                  </tr>
-                  <tr>
-                     <td>
-                        <small>{{ cupRatingGroupItem.dc_sum }}</small>
-                     </td>
-                  </tr>
-               </table>
-            </td>
-         </tr>
-      </tbody>
-   </table>
-   <table class="d-table d-sm-none table table-sm table-striped table-bordered shadow mb-0">
-      <thead class="thead-dark">
-         <tr>
-            <th scope="col" class="text-center align-middle">№</th>
-            <th scope="col" title="Ім'я Гравця" class="align-middle">Гравець</th>
-            <th scope="col" class="text-center align-middle" title="Забиті і Пропущені голи">З:П</th>
-            <th scope="col" class="text-center align-middle" title="Очки">О</th>
-         </tr>
-      </thead>
-      <tbody>
-         <tr
-            *ngFor="let cupRatingGroupItem of cupRatingGroup; let i = index"
-            [ngClass]="{
-               'table-primary': showPositionHighlight(i, 'config.cup.group.promotion'),
-               'table-info': showPositionHighlight(i, 'config.cup.group.possible_promotion'),
-               'table-secondary': showPositionHighlight(i, 'config.cup.group.other_competition')
-            }"
-         >
-            <th scope="row" class="text-center align-middle">{{ i + 1 }}.</th>
-            <td class="align-middle">
-               <ng-container [ngTemplateOutlet]="userInfo" [ngTemplateOutletContext]="{ item: cupRatingGroupItem }"></ng-container>
-            </td>
-            <td class="text-center align-middle">
-               <table align="center">
-                  <tr>
-                     <td>{{ cupRatingGroupItem.scored }}:{{ cupRatingGroupItem.missed }}</td>
-                  </tr>
-                  <tr>
-                     <td>
-                        <small class="text-muted">{{ cupRatingGroupItem.points_sum }}</small>
-                     </td>
-                  </tr>
-               </table>
-            </td>
-            <td class="text-center align-middle">
-               <strong>{{ cupRatingGroupItem.points }}</strong>
-            </td>
-         </tr>
-      </tbody>
-   </table>
+   <app-cup-rating-group-table [cupRatingGroup]="cupRatingGroup" [highlightConfig]="competition?.highlightConfig">
+   </app-cup-rating-group-table>
```

Comment:
Because of incorrect module declaration (`Core` instead of `CupModule`) this component is not shown. Please run the application to verify expected result after making any changes.